Cargo.toml

```diff
@@ -1,18 +1,18 @@
 [package]
-name = "sframe"
-version = "0.2.0"
 edition = "2021"
 license = "MIT/Apache-2.0"
+name = "sframe"
+version = "0.2.0"
```

Comment:
did you change the order here on purpose?

Reply:
Ouch, format on save. Will change that back